src/app/pages/cars/car/car.component.ts

```diff
@@ -43,7 +42,6 @@ export class CarComponent implements OnInit {
   public isCompanyCar!: AbstractControl;
   public carCatalogID!: AbstractControl;
   public carCatalog!: AbstractControl;
-  public converter!: AbstractControl;
   public converterType!: AbstractControl;
```

Comment:
The converterType should consist of at least 3 fields:
- converterWatts
- converterAmps
- converterNumberOfPhases
When drop down is selected, fill these 3 fields and save them to the database.

Reply:
You can also keep the converterType that will identify if it's the Standard, Option or Alternative (S, O or A)